out/main.go

```diff
@@ -4,19 +4,31 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"os"
 	"strconv"
+	"strings"
 
 	"github.com/arbourd/concourse-slack-alert-resource/concourse"
 	"github.com/arbourd/concourse-slack-alert-resource/slack"
 )
 
 func buildMessage(alert Alert, m concourse.BuildMetadata) *slack.Message {
 	fallback := fmt.Sprintf("%s -- %s", fmt.Sprintf("%s: %s/%s/%s", alert.Message, m.PipelineName, m.JobName, m.BuildName), m.URL)
+	log.Printf("TextFile location passed from yaml: %s", alert.TextFile)
+	const PutBasePath = "/tmp/build/put/"
```

Comment:
This can't be static. We need to be able to have some relative path. What if the file you are trying to pass is from a `get`?

Reply:
Hmm yea I see what you are saying, but arent we always in the put step for this resource? Any `output` from a previous step will will always go in the execution context of the step and in this case `/tmp/build/put/`? Unless we can pass a property from the build metadata that identifies where the file is passed from

Reply:
Ya me too! Let me look at this again 😅

Reply:
@arbourd After some digging and research I was able to confirm that `os.Args[1]` is indeed `/tmp/build/put/` I'll make the changes and push